### PR TITLE
Driver using API

### PR DESCRIPTION
**Pull request by luqasz**

### CODE REVIEW

**Comment by dbarrosop** [comment]
This is great! :)

I will do some tests with the vagrant box while we review the code.

Thanks for the hard work : )

**Comment by luqasz** [comment]
I'll rebase and push new version.

**Comment by luqasz** [comment]
I've rebased and all tests seems to pass.

File: napalm_ros/ros.py [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-    from io import StringIO
-#
-from napalm_base.base import NetworkDriver
-from napalm_base.exceptions import ConnectionException, MergeConfigException, CommandErrorException
-from rosapi import connect
+from napalm_base import NetworkDriver
 import napalm_base.utils.string_parsers
-import paramiko
-import mikoshell
+from napalm_base.helpers import (
```
Comment (dbarrosop) [inline napalm_ros/ros.py]: Please, don't do this. For easier readability do the following:

```
from napalm_base import helpers

...

helpers.ip(sadasd)
```

File: napalm_ros/utils.py [hunk 2]
```diff
@@ -1,183 +1,6 @@
...
-                        else:
-                            key_value['index'], key_value['flags'] = part_a.split(' ', 1)
-                    else:
-                        key_value['flags'] = part_a
-                else:
-                    key_value[key] = part_a
-                key = part_b
-        to_values_structured.append(key_value)
-    return to_values_structured
+from napalm_base.helpers import (
```
Comment (dbarrosop) [inline napalm_ros/utils.py]: same here

**Comment by luqasz** [comment]
Would you like to have all squashed or commit by commit ?

**Comment by dbarrosop** [comment]
It's alright, I will be looking into this today and most probably merging.

**Pull request merged**